apple-pie/styles/style.css

```diff
@@ -0,0 +1,61 @@
+body {
+    margin: 1rem;
+    padding: 0;
+    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 12px;
+    color: rgb(96, 96, 96);
+    max-width: 500px;
+    line-height: 1.1rem;
+    
+}
+main {
```

Comment:
after removing max-width, you need to change margin numbers to get good result ;)